src/cascadia/TerminalApp/App.cpp

```diff
@@ -29,6 +29,10 @@ namespace winrt::TerminalApp::implementation
 
     void App::Initialize()
     {
+        // LOAD BEARING
+        AddOtherProvider(winrt::Microsoft::Terminal::Control::XamlMetaDataProvider{});
+        AddOtherProvider(winrt::Microsoft::UI::Xaml::XamlTypeInfo::XamlControlsXamlMetaDataProvider{});

```

Comment:
There seems to only be one call of `AddOtherProvider` (happens on line 114 in this PR), why do we need both of these then?